pycon/settings.py

```diff
@@ -151,7 +150,7 @@ def _(x):
 )
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = config('SECRET_KEY', default='your-secret-key')
+SECRET_KEY = config('SECRET_KEY')
```

Comment:
This should make it harder to deploy in any environment with an insecure secret key.